code/modules/mob/living/carbon/human/species.dm

```diff
@@ -294,12 +316,33 @@
 					S = frills_list[H.dna.features["frills"]]
 				if("horns")
 					S = horns_list[H.dna.features["horns"]]
+				if("ears")
+					S = ears_list[H.dna.features["ears"]]
 				if("body_markings")
 					S = body_markings_list[H.dna.features["body_markings"]]
 
 			if(!S || S.icon_state == "none")
 				continue
+
+			//A little rename so we don't have to use tail_lizard or tail_human when naming the sprites.
+			if(bodypart == "tail_lizard" || bodypart == "tail_human")
+				bodypart = "tail"
+			else if(bodypart == "waggingtail_lizard" || bodypart == "waggingtail_human")
+				bodypart = "waggingtail"
```

Comment:
Making content admin-only is terrible imo

Just leads to favouritism and self-serving.

On 1 July 2015 at 04:04, Cheridan <notifications@github.com> wrote:

> In code/modules/mob/living/carbon/human/species.dm
> <https://github.com/tgstation/-tg-station/pull/10209#discussion_r33590112>
> :
>
> >  				if("body_markings")
> >  					S = body_markings_list[H.dna.features["body_markings"]]
> >
> >  			if(!S || S.icon_state == "none")
> >  				continue
> > +
> > +			//A little rename so we don't have to use tail_lizard or tail_human when naming the sprites.
> > +			if(bodypart == "tail_lizard" || bodypart == "tail_human")
> > +				bodypart = "tail"
> > +			else if(bodypart == "waggingtail_lizard" || bodypart == "waggingtail_human")
> > +				bodypart = "waggingtail"
>
> Ah, ok.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/10209/files#r33590112>.
>